tests/test_signing.py

```diff
@@ -120,7 +129,9 @@ def test_sign_all_in_repodata(request):
 
     # Make sure there is one signature entry for every artifact entry, and no
     # mystery entries.
-    assert repodata["packages"].keys() == repodata_signed["signatures"].keys()
+    assert set(repodata["packages"].keys()) | set(
+        repodata["packages.conda"].keys()
+    ) == set(repodata_signed["signatures"].keys())
```

Comment:
does this mean v1 and v2 packages have identical signatures?

```suggestion
    assert {
        *repodata["packages"].keys(),
        *repodata["packages.conda"].keys(),
    } == set(repodata_signed["signatures"].keys())
```